independent-publisher-connectors/Electricity Maps/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1239 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "You can use Electricity Maps API to get access to information about where the electricity in a specific area comes from, how much carbon was emitted to produce it, and to get historical data, live data and forecasts.",
+    "title": "Electricity Maps",
+    "version": "0.1",
+    "contact": {
+      "name": "Vitalii Sorokin",
+      "url": "https://www.electricitymaps.com/",
+      "email": "sorokindva@gmail.com"
+    }
+  },
+  "host": "api-access.electricitymaps.com",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/carbon-intensity/forecast": {
+      "get": {
+        "parameters": [
+          {
+            "name": "zone",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "A string representing the zone identifier",
```

Comment:
@VitaliiSorokin all parameters' descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters)
Please fix this in all parameters in the swagger file